app/assets/javascripts/userdash.js

```diff
@@ -29,6 +29,11 @@ require(["lib/models", "lib/tools", "knockout", "sammy"], function (models, tool
             self.isLoading(false);
         };
 
+        self.setInfo = function (level, text) {
```

Comment:
This solution is fine for now, but in the long run userdash should import and use the `MessageBox` from `lib/msg` in the long run.